<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fillerTab</class>
 <widget class="QWidget" name="fillerTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,2">
   <item>
    <widget class="QGroupBox" name="canvasGroup">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="title">
      <string>холст</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tableGroup">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="title">
      <string>Инспектор</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="fillPointGroup">
        <property name="title">
         <string>Затравочный пиксел</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="fillXInput">
             <property name="maximum">
              <number>3000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="fillYInput">
             <property name="maximum">
              <number>3000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="selectFillPointButton">
           <property name="text">
            <string>выбрать мышкой</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1000</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="fillTime">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Время заполнения: --</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>ЛКМ - рисование контура</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="colorForm">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>цвет</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="fillButton">
        <property name="text">
         <string>заполнить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fillStepButton">
        <property name="text">
         <string>заполнить с задержкой</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearFillButton">
        <property name="text">
         <string>очистить заполнение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeRegionButton">
        <property name="text">
         <string>удалить область</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>очистить холст</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
